unittests/message_handlers/test_service_msg_handler.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from message_handlers.service_msg_handler import ServiceMsgHandler
+
+
+class TestServiceMsgHandler(unittest.TestCase):
+    service_info = {
+            'service_name': 'kafka',
+            'status': 'failed'
+        }
+
+    def setUp(self) -> None:
+        self.service_msg_handler = ServiceMsgHandler()
+        self.service_msg_handler.cluster_id = 1
+        self.service_msg_handler.site_id = 1
+        self.service_msg_handler.rack_id = 1
+        self.service_msg_handler.node_id = 1
+        self.service_msg_handler.storage_set_id = 1
+        self.service_msg_handler.host_id = 1
+
+    def test_create_actuator_response(self):
+        msg = self.service_msg_handler._create_actuator_response(self.service_info)
+        assert msg['host_id'] == 1
+        assert msg['instance_id'] == 'kafka'
```

Comment:
As discussed, framework wise looks fine to start with.

For this test, add more useful asserts.

Reply:
added more asserts.